### PR TITLE
LPS-189998 Replace alert divs with clay:alert in rss-web and trash-web

### PR DESCRIPTION
**Pull request by DiegoHu97**
[Link to ticket](https://github.com/liferay-echo/liferay-portal/pull/12816)

## Test scenarios

**Case 1**
- Go to Site Builder > Pages
- Create a content page > add a RSS Publisher widget to it > see alert 
Screenshot 2023-07-10 at 21 30 00

**Case 2**
- Go to Site Builder > Pages
- Create a content page > add a RSS Publisher widget to it 
- Go to the widget configuration > Add a title and the following url: https://www.liferay.com/es/home
- Save the configuration > see alert 
Screenshot 2023-07-11 at 15 23 28

**Case 3**
- Go to Content & Data > Web content
- Create a folder
- Create a web content inside the folder
- Delete the Web content 
- Delete the folder
- Go to Recycle bin
- Delete the folder
- Restore the Web content > see modal 
Screenshot 2023-07-11 at 15 20 57

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:relevant

File: modules/apps/trash/trash-web/src/main/resources/META-INF/resources/view_container_model.jsp [hunk 1]
```diff
@@ -20,9 +20,9 @@
 TrashContainerModelDisplayContext trashContainerModelDisplayContext = new TrashContainerModelDisplayContext(liferayPortletRequest, liferayPortletResponse);
 %>
 
-<div class="alert alert-block">
-	<liferay-ui:message arguments="<%= trashContainerModelDisplayContext.getMissingContainerMessageArguments() %>" key="the-original-x-does-not-exist-anymore" translateArguments="<%= false %>" />
-</div>
+<p class="p-3">
```
Comment (victorg1991) [inline modules/apps/trash/trash-web/src/main/resources/META-INF/resources/view_container_model.jsp:23]: Shouldn't this be an alert?

File: modules/apps/rss/rss-web/src/main/resources/META-INF/resources/feed.jspf [hunk 2]
```diff
@@ -114,5 +114,12 @@
 		<div class="alert alert-danger">
 			<liferay-ui:message arguments="<%= HtmlUtil.escape(rssFeed.getURL()) %>" key="cannot-be-found" translateArguments="<%= false %>" />
 		</div>
+
+		<clay:alert
+			displayType="danger"
+			id='<%= liferayPortletResponse.getNamespace() + "message" %>'
+		>
+			<liferay-ui:message arguments="<%= HtmlUtil.escape(rssFeed.getURL()) %>" key="cannot-be-found" translateArguments="<%= false %>" />
+		</clay:alert>
```
Comment (victorg1991) [inline modules/apps/rss/rss-web/src/main/resources/META-INF/resources/feed.jspf:123]: We probably need to remove the previous div, isn't it? :p

**Comment by victorg1991** [comment]
@DiegoHu97 Just a couple of comments 😇

**Comment by Tim-Cao** [comment]
Hi @DiegoHu97, the Home folder is invisible on Select Restore Folder modal.
image
Step to reproduce

1. Add a new site
2. Navigate to the Documents and Media admin
3. Add a new folder
4. Add three documents inside folder
5. Move the folder to Recycle Bin
6. Navigate to the Recycle Bin
7. Go into the folder
8. Restore any document

**Comment by Tim-Cao** [comment]
Btw, I updated the Poshi side in https://github.com/liferay-echo/liferay-portal/pull/12842 @DiegoHu97

**Pull request closed without merging**